README.md

```
@@ -11,7 +11,7 @@ From-scratch Python implementations of Gaussian elimination, QR decomposition, a
 - [Setup](#setup)
 - [Parts](#parts)
   - [Part 1 — Gaussian Elimination](#part-1--gaussian-elimination)
-  - [Part 2 — QR Decomposition + Manim Video](#part-2--qr-decomposition--manim-video)
+  - [Part 2 — SVD Decomposition + Manim Video](#part-2--qr-decomposition--manim-video)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)